src/core/public/core_system.ts

```diff
@@ -323,6 +340,26 @@ export class CoreSystem {
           type: 'long',
           _meta: { description: 'The used size of the heap', optional: true },
         },
+        load_start: {
+          type: 'long',
+          _meta: { description: 'When the render template starts loading assets', optional: true },
+        },
+        bootstrap: {
+          type: 'long',
+          _meta: { description: 'When kbnBootstrap callback is called', optional: true },
+        },
+        core_create: {
```

Comment:
nit: how do you feel about calling it `core_created`? I initially thought it was when the constructor was called, but looking at the description and implementation, it's actually when the constructor call is completed.